Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN bash -c "echo 'deb http://us.archive.ubuntu.com/ubuntu focal main restricted universe' | tee /etc/apt/sources.list; apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3B4FE6ACC0B21F32; apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 871920D1991BC93C; apt update"
+
+RUN apt install -y gnupg2 curl procps libpq-dev
+
+RUN DEBIAN_FRONTEND="noninteractive" bash ./setup.sh -a
```

Comment:
Postgres should not be required for this container, and bundler always comes preinstalled with the ruby docker container.

Additionally, I would like to build this image on Dockerhub so that users don't have to build it every time, we will most likely need to break it out into 2 steps:

1. `docker-compose up -d`
1. Run `docker-compose run web rails db:create_admin`

This way `docker-compose` can just pull the image, and then the user can run the 1 time command to create their administrator user separately.

Reply:
Since libpq-dev is required to build the postgresql gem and it's not installed by default, it's installed here. 

My plan here now is to to have the use do ``docker-compose up -d`` and then ``docker-compose run web bash ./setup.sh -s`` to load the database and create an admin user using the same script.